WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -31,6 +31,27 @@
           </ul>
         <% end %>
       </li>
+      <li>
+        <%= button_to competition_cancel_path, class: 'btn btn-danger', disabled: !@competition.can_be_cancelled?, method: :post, data: { confirm: t('competitions.cancel_confirm') } do %>
+          <%= t 'competitions.cancel' %>
+        <% end %>
+        <% if !@competition.can_be_cancelled? && !@competition.cancelled? %>
+          Note: only confirmed competitions can be cancelled.
```

Comment:
This is admin section, so English is just fine, it's just a little inconsistent with the other phrases here, which are translated. Just pointing out to ensure it's expected.

Reply:
Good point; I think I must have missed these string as I usually do the localization last when making changes!